vividus-plugin-visual/src/main/java/org/vividus/visual/engine/VisualTestingEngine.java

```diff
@@ -34,28 +33,28 @@
 import ru.yandex.qatools.ashot.Screenshot;
 import ru.yandex.qatools.ashot.comparison.ImageDiff;
 import ru.yandex.qatools.ashot.comparison.ImageDiffer;
-import ru.yandex.qatools.ashot.comparison.PointsMarkupPolicy;
 import ru.yandex.qatools.ashot.util.ImageTool;
 
-@SuppressWarnings("MagicNumber")
 public class VisualTestingEngine implements IVisualTestingEngine
 {
     private static final Logger LOGGER = LoggerFactory.getLogger(VisualTestingEngine.class);
-
-    private static final Color DIFF_COLOR = new Color(238, 111, 238);
+    private static final int ONE_HUNDRED = 100;
+    private static final int SCALE = 3;
 
     private final ScreenshotProvider screenshotProvider;
     private final IBaselineRepository baselineRepository;
+    private final DiffMarkupPolicyFactory diffMarkUpPolicyFactory;
```

Comment:
```suggestion
    private final DiffMarkupPolicyFactory diffMarkupPolicyFactory;
```
please fix all such references